documentation/asciidoc/topics/yaml/prometheus_service_monitor.yaml

```diff
@@ -3,26 +3,27 @@ kind: ServiceMonitor
 metadata:
   labels:
     k8s-app: prometheus
-  name: datagrid-monitoring <1>
-  namespace: infinispan-monitoring <2>
+  name: infinispan-monitoring # Names the service monitor.
+  namespace: infinispan-monitoring # Namespace for the service monitor.
```

Comment:
Can we default to cluster namespace rather then subconsciously suggesting to create `ServiceMonitor` in different one? At least it's tested when created in the same and given the example is always able to monitor single cluster only then there is no need to create it in another namespace.

Reply:
It would be nice to implement #959 and remove this options from the manual user steps